source/reference/template-apps.txt

```diff
@@ -97,13 +97,13 @@ with services and front-end code. You can:
 
 2. Follow on-screen guides to customize your app.
 
-3. Use the {+cli-bin+} to download the code for your app so you can build the frontend
-   (if one is included) and interact with the App through the cli.
+3. Use the {+cli-bin+} to download the code for your app so you can build the front end
```

Comment:
```suggestion
3. Use the {+cli-bin+} to download the code for your app so you can build the frontend
```

i think generally we do as 1 word.

Reply:
[style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#f) has it as two (hyphenated if an adjective), which is why I updated it.